webrtc/simplecall.html

```diff
@@ -97,9 +99,11 @@
     gFirstConnection.addIceCandidate(event.candidate);
   });
 
-  var onRemoteStream = test.step_func(function(event) {
+  var onRemoteTrack = test.step_func(function(event) {
     var videoTag = document.getElementById('remote-view');
-    videoTag.srcObject = event.stream;
+    if (!videoTag.srcObject) {
```

Comment:
I guess that multiple track events are fired, but only one addstream event is fired. Makes sense.